arrow-core-data/src/main/kotlin/arrow/core/Either.kt

```diff
@@ -803,9 +803,21 @@ sealed class Either<out A, out B> : EitherOf<A, B> {
    * Left(12).toOption()  // Result: None
    * ```
    */
+  @Deprecated("Deprecated, use `orNull` instead", ReplaceWith("orNull()"))
   fun toOption(): Option<B> =
     fold({ None }, { Some(it) })
 
+  /**
+   * Returns the right value if it exists, otherwise null
+   *
+   * Example:
+   * ```
```

Comment:
We annotate these with `kotlin:ank:playground` that makes sure that they're compiled checked when building Arrow. So that when a user copies a snippet from the docs, it's guaranteed to work for said version.
That's done by `kotlin:ank`.

And `playground`, adds some additional metadata that's need to turn them into runnable snippets on the Arrow doc website :)

Reference to the tool we use: https://github.com/arrow-kt/arrow-ank